docs/source/format/Integration.rst

```diff
@@ -20,8 +20,92 @@
 Integration Testing
 ===================
 
+Our strategy for integration testing between Arrow implementations is:
+
+* Test datasets are specified in a custom human-readable, JSON-based format
+  designed for Arrow
```

Comment:
For clarify change "designed for Arrow" to "designed exclusively for Arrow's integration tests" to stave off questions of whether this is intended as a general JSON representation of Arrow data